source/blog/2016-01-14-atomic-app-0-3-1.html.md

```diff
@@ -0,0 +1,145 @@
+          accessMode: "ReadWrite"
+          size: 4
+```
+
+And running it:
+```bash
+docker build -t projectatomic/helloapache-ps
+atomic run projectatomic/helloapache-ps --provider=docker
+```
+
+When you deploy your application, Atomic App will automatically take the `persistentVolume` claim under requirements and deploy it to it's respective provider. In the case of Kubernetes, a [persistent volume claim](https://github.com/kubernetes/kubernetes/blob/master/docs/design/persistent-storage.md) will be defined before proceeding in launching a [replication controller](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/replication-controller.md) or [pod](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/pods.md).
+
+__OpenShift stop__
+
+Compatibility for stopping deployed Atomic App containers on OpenShift has now been added.
+
+```
+atomic stop projectatomic/helloapache /var/lib/atomicapp/yournuleculedir
+```
+
+__HTTPS (TLS) support for OpenShift__
+
+With version 0.3.1 of Atomic App we enable https/tls support by default. 
+
+Certificates are validated against the Mozilla CA Certificate List using the [certifi](https://certifi.io/en/latest/) project.
+
+Both options can be set using `answers.conf` file or via command line options.
+
+  * `providercafile` - If your https API endpoint isn't signed by trusted CA. You can use this option to provide path to your CA certificate.
+  * `providertlsverify` - By setting this to False you can disable TLS/SSL verification. (default is True)
+
+Example answers.conf:
+```
+[general]
+provider = openshift
+providerapi = https://127.0.0.1:8443
+accesstoken = sadfasdfasfasfdasfasfasdfsafasfd
+namespace = mynamespace
+providertlsverify = True
+providercafile = /etc/myca/ca.pem
+```
+
+Example of disabling verification using command line argument:
+```
+atomic run myapp --provider=openshift --providertlsverify=False
+```
+
+When using [native execution](https://github.com/kadel/atomicapp/blob/master/docs/providers/openshift/overview_native.md) (`oc new-app`) to run Nulecule application on OpenShift you can pass command line arguments via environment variables `ATOMICAPP_APPEND_ARGS`.
+
+```
+oc new-app myapp --grant-install-rights -e ATOMICAPP_APPEND_ARGS='--providertlsverify=False'
+```
+
+__Nested Nulecule support for OpenShift__
+
+Native support for launching Nulecules within OpenShift has been added. You can now pull from a library of Atomic App's (Nuleculized applications) that exist in the Docker hub and launch them into OpenShift.
+
+Using the [guestbookgo-atomicapp](https://github.com/projectatomic/nulecule-library/tree/master/guestbookgo-atomicapp) example you can now start the container via the `oc new-app` command.
+
+```
+oc new-app projectatomic/guestbookgo-atomicapp --grant-install-rights
+```
+
+Atomic App will now evaluate what environment the container is being ran from and communicate with OpenShift in order to bring up the Nuleculized application.
```

Comment:
I would just delete the line below and add it as a small item onto this paragraph:

```
To see a more complete example of running a Nulecule via `oc new-app` see Dusty Mabe's [post](http://dustymabe.com/2016/01/18/running-nulecules-in-openshift-via-oc-new-app/) on the subject.
```